aaudio/.google/packaging.yaml

```diff
@@ -0,0 +1,7 @@
+status:       PUBLISHED
+technologies: [Android, NDK]
+categories:   [NDK]
+languages:    [C++, Java]
```

Comment:
this is good one, it is NOT intended to be Java sample at all: there are much better ones for that purpose.  Took it out ( it was copy-paste thing )